aws-opentelemetry-distro/src/amazon/opentelemetry/distro/_aws_metric_attribute_generator.py

```diff
@@ -407,6 +409,9 @@ def _set_remote_type_and_identifier(span: ReadableSpan, attributes: BoundedAttri
         elif is_key_present(span, GEN_AI_REQUEST_MODEL):
             remote_resource_type = _NORMALIZED_BEDROCK_SERVICE_NAME + "::Model"
             remote_resource_identifier = _escape_delimiters(span.attributes.get(GEN_AI_REQUEST_MODEL))
+        elif is_key_present(span, AWS_SNS_TOPIC_ARN):
+            remote_resource_type = _NORMALIZED_SNS_SERVICE_NAME + "::Topic"
+            remote_resource_identifier = _escape_delimiters(span.attributes.get(AWS_SNS_TOPIC_ARN))
```

Comment:
Please add TODO here that we'll need to parse the actual resource name from the long ARN string by following the rules defined in https://quip-amazon.com/ShMUAP1okoxs/Support-AWS-Resources-with-Different-CFN-Identifier-Formats#temp:C:IHL175110d2baee4144bbe0f54cc

Reply:
we can create a new attribute named `aws.cloudformation.primary.identifier` to store the orignal CFN prmiray identifier value from OTel instrumentation.

Reply:
can we use `aws.remote.resource.cfn.primary.identifier` and the new attribute name in EMF will be `RemoteResourceCfnPrimaryIdentifier`?